spring-boot-admin-starter-client/src/test/java/de/codecentric/boot/admin/services/ApplicationFactoryTest.java

```diff
@@ -24,7 +25,8 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
 
-public class AdminClientPropertiesTest {
+//FIXME: do test on methods with visibility public, but not protected ...
```

Comment:
Yeah the test should be all about the public methods. Should be no problem to test the behaviour of the private methods via the public api....